debian/control

```diff
@@ -5,7 +5,7 @@ Priority: optional
 Build-Depends: automake, autoconf, libtool, libssl-dev, sed,
  make, coreutils, pkg-config, gawk, sed, bash, dh-exec
 
-Package: libtpms
+Package: libtpms0
```

Comment:
With this change you also have to rename debian/libtpms libtpms.install to libtpms0.install.

Reply:
Are you going to make these changes? If not, I will do them myself on Thursday.